src/pageObject.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return `Conduit logo`;
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the Conduit logo`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, header, slug) {
```

Comment:
you don't need to pass parameters for `header` as this property only inherits the `Header` class